projects/shepherd/src/lib/shepherd.service.ts

```diff
@@ -42,6 +42,13 @@ export class ShepherdService {
   complete() {
     this.tourObject.complete();
   }
+  
+  /**
+   * Gets the current step
+   */
+  getCurrentStep() {
```

Comment:
What's the reasoning behind adding this method? You can already call `this.tourObject.getCurrentStep();` right?